docs/Sprint-Review/sprint-97-summary.md

```diff
@@ -0,0 +1,66 @@
+  * [#2814 Aggregate Cloud.gov ES instances](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2814)
+  * [#2818 As tech lead, I need TDP to reject files that do not have an update indicator == D](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2818)
+  * [#2839 Spike - Cloud.gov Kibana Reschedule Route Issue](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2839)
+  * [#2846 [Design Deliverable] Submission History Knowledge Center Explainer](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2846)
+  * [#2908 As a DIGIT Team user, I need the ability to export parsed data from DAC to csv](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2908)
+
+### Ready to Merge
+
+
+### Submitted (QASP Review, OCIO Review)
+  * [#1345 [Design Deliverable] Email Template for Data Due Date Reminder](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/1345)
+  * [#2509 As a data analyst I need to know when my data has been processed with or w/o errors](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2509)
+  * [#2826 As tech lead, I need some record types that currently require trailing spaces to be parsed](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2826)
+  * [#2842 Complete cat4 validation implementation](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2842)
+  * [#2847 [Design Deliverable] Error Report Knowledge Center Explainer](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2847)
+  * [#2884 Generate preparser errors when multi-record rows are the wrong length or are missing space-filled second records - M3 and Tribal T3](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2884)
+  * [#2897 As a data analyst I want finalized language and guidance resources in Submission History & Error Reports](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2897)
+
+### Closed (not merged)
+* [#2757 Generate preparser errors when multi-record rows are the wrong length or are missing space-filled second records](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2802)
+
+---
+
+## Moved to Next Sprint (In Progress, Blocked, Raft Review)
+### In Progress
+  *	[#2802 [Django A11Y] [Spike] Multiple filter option](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2802)
```

Comment:
@victoriaatraft might have gotten on here accidentally? I'm showing this as having entered in progress 2 days ago

Reply:
Interesting @reitermb #2802 shows it was moved to Raft Review on the [Sprint Report](https://github.com/raft-tech/TANF-app#workspaces/sprint-board-5f18ab06dfd91c000f7e682e/reports/burndown?milestoneId=Z2lkOi8vcmFwdG9yL1NwcmludC8xNjk2NDI0&showPRs=false).